src/components/playback/playbackmanager.js

```diff
@@ -2039,6 +2045,20 @@ class PlaybackManager {
             return player.play(options);
         }
 
+        const getAdditionalParts = async (items) => {
+            const getOneAdditionalPart = async function (item) {
+                if (item.Type === 'Movie') {
+                    const client = ServerConnections.getApiClient(item.ServerId);
+                    const user = await client.getCurrentUser();
+                    const additionalParts = await client.getAdditionalVideoParts(user.Id, item.Id);
+                    return (additionalParts.Items.length) ? [item, ...additionalParts.Items] : [item];
```

Comment:
Does this work?
```suggestion
                    if (additionalParts.Items.length) return [item, ...additionalParts.Items];
```
^ one exit for simple items.